### ISSUE TITLE
Engine license script should work on Mac/Windows

### ISSUE THREAD

**Issue by cbracken** [flutter/flutter#28267]
Currently the engine license script produces different output on Linux, Mac, and Windows, but we treat the Linux output as canonical.

The reason for the differences has to do with the differing paths of third-party dependencies depending on host OS (e.g. on Linux, we install the Android NDK, on Mac, we install a Mac NDK, with platform-specific differences in contents).

Downloading the Linux contents on Mac/Windows would work, but isn't a reasonable solution. It would be useful to catalogue the differences to see how extensive they are and whether they could be abstracted out without risking correctness.

**Comment by pedromassangocode** [comment]
Is this still a valid issue today?

**Comment by cbracken** [comment]
Yes this is still an issue, unfortunately. It's fairly low priority, but something that would be nice to fix eventually.
